app/controllers/concerns/pages.rb

```diff
@@ -0,0 +1,39 @@
+module Pages
+  extend ActiveSupport::Concern
+
+  included do
+    resource_actions :default
+
+    schema_type :strong_params
+
+    allowed_params :create, :url_key, :title, :content, :language
+    allowed_params :update, :url_key, :title, :content, :language
+
+    before_filter :set_language_from_header, only: [:index]
+
+    define_method(:link_header) do |resource|
```

Comment:
If those methods are just defined normally in the concern, they're pulled in but then overridden later by the creatable_resource lib. If they're defined in the included block they're the ones doing the overriding, which is what I want (and expected in the first place). I chalked it up to just order-of-operations weirdness in the hierarchy of includes and inheritance.